src/js/base/ffi-helpers.js

```diff
@@ -456,6 +456,8 @@ define(["js/runtime-util", "trove/lists", "trove/sets", "trove/option", "trove/e
 
         makeList: makeList,
         makeTreeSet: makeTreeSet,
+
+        isOption: runtime.getField(O, "is-Option"),
```

Comment:
Agree that all other functions are raw JS functions, so this seems to be inconsistent. However, most values that you would `cases` on would be constructed from `data` which normally would come from Pyret. I guess it is also awkward to write `.app` everytime you write `ffi.case`.